docs/faq.md

```diff
@@ -0,0 +1,84 @@
+# Frequently Asked Questions
+
+## How do I customize X on the worker group's settings?
+
+All the options that can be customized for worker groups are listed in [local.tf](https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/local.tf) under `workers_group_defaults_defaults`.
+
+Please open Issues or PRs if you think something is missing.
+
+## Why are nodes not being registered?
+
+This is usually caused by a networking or endpoint configuration issue.
+
+At least one of the public or private endpoints must be enabled in order for access to the cluster to work.
+
+Your nodes need to be able to contact the EKS endpoint. By default the module only creates a public endpoint. You should also enable the private endpoint by setting `cluster_endpoint_private_access = true` on this module.
+
+If you have the private endpoint enabled ensure that you also have VPC DNS enabled. Set `enable_dns_hostnames = true` on your `aws_vpc` resource or the `terraform-aws-module/vpc/aws` community module.
+
+Nodes need to be able to connect to AWS services plus pull down container images from repos. You can either:
+- enable endpoints to the relevant services, if only using ECR repos and for some reason cannot enable public outbound: EC2 API, ECR API, ECR DKR and S3
+- enable outbound public internet access:
+  - Private subnets: via a NAT gateway or instance
```

Comment:
Different indentation here 😃

Reply:
Yes, it's a sub list. It makes more sense if you look at the rendered output:
https://github.com/dpiddockcmp/terraform-aws-eks/blob/faq/docs/faq.md#why-are-nodes-not-being-registered

- enable outbound public internet access:
  - Private subnets: via a NAT gateway or instance
  - Public subnets: assign public IPs to nodes